### DISCUSSION TITLE
Hub

### DISCUSSION THREAD

**Discussion by GvidelisGaidelis** [Courseplay/Courseplay_FS22#1752] [Ideas]
So FS22 has a new hub, but it's not very handy for me, old one was better (it could be just for me) so maybe there is the way somehow to get it back? Maybe it's comming soon, I don't know.

**Comment by schwiti6190** [comment]
@GvidelisGaidelis The hud will mostly stay, as it is now. If you've got any feedback what could be optimized then let us now.
The main driving force behind the change was the integration of the in game menu pages( almost complete gamepad support) and the simplicity for new players.
The old hud wasn't great for new player, as even myself, as a developer didn't figure out all the possible settings/features in the old FS 19 hud :)

**Reply by GvidelisGaidelis** [reply]
Ok, thx for feedback. <3

**Comment by Tensuko** [comment]
The old hud wont come back at all.

**Reply by GvidelisGaidelis** [reply]
Sad :(

**Comment by Leon83858** [comment]
@Tensuko you dont want new descusions about this fine then send me this earlyer also i realy like how you ignored basicaly everything i said 

i slowly learn how it works but im still missing the funktion of telling the course generator where to set the 1st WP

**Reply by Tensuko** [reply]
Read the ingame help menu.

**Reply by Leon83858** [reply]
i do but the pictures showing where to find what is described re tiny & not readable on a 24" monitor

**Reply by Tensuko** [reply]
You have to read the text and then go into the menu then you will understand it....
